docs/administration-guide/gaffer-config/config.md

```diff
@@ -0,0 +1,150 @@
+## How to configure a Gaffer instance
+
+First you must set up the files and directories you will need for the instance. As it stands there
+are a couple of different ways to run a Gaffer project this example will use a logical structure
+that suites a stand alone deployment consisting of the following file structure:
+
+!!! example "Example Gaffer project structure"
+
+    !!! tip ""
+        Click the plus symbols for a brief description of each file
+
+    ```yaml
+    ├── config
+    │   ├── gaffer
+    │       ├── application.properties #(1)!
+    │       ├── data                   #(2)!
+    │       │   ├── neo4jExport.csv
+    │       ├── graph
+    │       │   └── graphConfig.json #(3)!
+    │       ├── schema
+    │       │   ├── elements.json #(4)!
+    │       │   └── types.json    #(5)!
+    │       └── store
+    │           ├── operationsDeclarations.json #(6)!
+    │           └── store.properties            #(7)!
+    │   
+    └── docker-compose.yaml #(8)!
+    ```
+
+    1. Properties file that generally sets the file locations of other Gaffer
+        configs e.g. schemas (note these are the absolute paths inside the
+        container).
+    2. Any data files, e.g. CSV, to be made available to the Gaffer container.
+    3. The main graph config file to set various properties of the overall graph.
+    4. This file holds the schema outlining the elements in the graph, e.g. the
+    nodes (aka entities) and edges.
+    5. This file defines the different data types in the graph and how they are
```

Comment:
These could probably point to schema documentation

Reply:
done